cmd/reports-controller/main.go

```diff
@@ -197,6 +202,7 @@ func main() {
 		omitEvents                       string
 		skipResourceFilters              bool
 		maxAPICallResponseLength         int64
+		maxBackgroundReports             int
```

Comment:
Don't we need to initialize this, as done here:

https://github.com/kyverno/kyverno/blob/db7584c4171dab8b57da0b7acc89e3ce3eab34b3/cmd/background-controller/main.go#L111

Reply:
yeah you are right, must have missed it 
I have added it now